locales/en.yaml

```diff
@@ -77,6 +77,17 @@ wiki:
       TAGS: Tags
       BREADCRUMBS: Breadcrumbs
 
+email:
+  GOTO_SITE: Go to website
+  SUBSCRIBED: You received this email because you are subscribed.
+  UNSUBSCRIBE: Unsubscribe
+  UNSUBSCRIBE_TEXT: "If you wish to unsubscribe, go here:"
+
+  social:
+    BLOG: Wikijump Blog
+    TWITTER: "@GetWikijump"
+    GITHUB: scpwiki/wikijump
```

Comment:
Should this be a translation thing? It seems like you'd want this to be customized per-install, so I guess in `web/config` instead.

Like, looking at how these are used, it seems better to have this be the names of the platforms instead ("Twitter", etc).

Reply:
Yeah, I've been conflicted on how this should be handled... If this is configured (by not editing the templates, that is), then I guess I would do something like:
```php
$social = [];
$social[] = ['type' => 'web', 'name' => 'Wikijump Blog'];
$social[] = ['type' => 'twitter', 'name' => '@GetWikijump'];
// and so on
```
and put that in a `web/config` file, like you said. I wish I could type arrays...

Reply:
You don't need to push individually like that:
```php
$social = [
    ['type' => 'web', 'name' => 'Wikijump Blog'],
    ['type' => 'twitter', 'name' => '@GetWikijump'],
    // ...
];
```

Reply:
But yeah it's not a clear cut case, I agree

Reply:
I only used the syntax because I was lazy when writing the reply lol.

What do you think of this in `web/config/wikijump.php`?
```php
/**
 * Social media links that appear in the footer of emails.
 *
 * Each array element is an array with the following keys:
 * - name: the name of the social media service (MJML syntax)
 * - url: the URL to link to
 * - text: the text to display
 * - src: icon image source
 *
 * None of the keys are required.
 */
'mail_social_links' => [
    [
        'name' => 'web',
        'url' => 'https://www.wikijump.org',
        'text' => 'Wikijump Blog',
    ],
    [
        'name' => 'twitter',
        'url' => 'https://twitter.com/getwikijump',
        'text' => 'Twitter',
    ],
    [
        'name' => 'github',
        'url' => 'https://github.com/scpwiki/wikijump',
        'text' => 'GitHub',
    ],
],
```

Reply:
Okay, I've changed it so that the social links are configurable via `web/config/wikijump.php`.

Reply:
Looks fine to me.